src/schema/v2/fair.ts

```diff
@@ -262,35 +266,48 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
             type: GraphQLBoolean,
             defaultValue: false,
           },
+          page: {
+            type: GraphQLInt,
+            defaultValue: 1,
```

Comment:
I noticed we often don't specify a default page number - I'm not really sure why (or if there are potential ramifications), so I started a thread about it in [#dev-help](https://artsy.slack.com/archives/CP9P4KR35/p1626703855113200). Let's see if there are any best practices we should be following!

Reply:
[Seems that there is no significant difference](https://artsy.slack.com/archives/CP9P4KR35/p1626705184113800?thread_ts=1626703855.113200&cid=CP9P4KR35) so I merge